pallets/ddc-clusters/src/cluster.rs

```diff
@@ -31,6 +33,26 @@ pub struct ClusterParams<AccountId> {
 	pub node_provider_auth_contract: AccountId,
 }
 
+// ClusterGovParams includes Governance sensetive parameters
+#[derive(Clone, Encode, Decode, RuntimeDebug, TypeInfo, PartialEq)]
+pub struct ClusterGovParams<Balance> {
+	pub treasury_share: Perbill,
+	pub validators_share: Perbill,
+	pub cluster_reserve_share: Perbill,
+	#[codec(compact)]
+	pub edge_bond_size: Balance,
```

Comment:
Let's start a smooth migration from `edge_` prefix to `cdn_` prefix for everything that is CDN-node related. 
Let's not do this renaming in the Staking Pallet for now, as we will break the client-side SDK and apps that depend on it, but let's at least not use the `edge_` prefix for new fields/endpoints and use the `cdn_`